libc/src/__support/FPUtil/aarch64/FEnvImpl.h

```diff
@@ -17,10 +17,10 @@
 #endif
 
 #include <arm_acle.h>
-#include <fenv.h>
 #include <stdint.h>
 
 #include "src/__support/FPUtil/FPBits.h"
+#include <fenv.h>
```

Comment:
Keep `#include <fenv.h>` in where it was, and add `#include "hdr/fenv_macros.h` here also.  It seems redundant, but it will make the next step easier.

Reply:
This line should not be moved, please move it back.

Reply:
actually, most files remove the include of `fenv.h` entirely. Any idea what we're keeping it around for here?